libraries/ui-library/src/components/six-error/six-error.scss

```diff
@@ -0,0 +1,5 @@
+.six-error-container {
+  text-align: left;
+  color: var(--six-color-blue);
```

Comment:
`six-color-blue` seems wrong

Reply:
This was a debugging thing that has no effect, will remove